### ISSUE TITLE
Update "proof.proofPurpose" test to check if the string maps to a URL.

### ISSUE THREAD

**Issue by JSAssassin** [w3c-ccg/data-integrity-test-suite-assertion#19]
Normative statement in the spec [Verifiable Credential Data Integrity 1.0](https://www.w3.org/TR/vc-data-integrity/) published 21 Oct 2023 - The reason the proof was created MUST be specified as a string that maps to a URL.

The current test only checks if "proof.proofPurpose" exists and if it is a string and does not test if the string maps to a URL. https://github.com/w3c-ccg/data-integrity-test-suite-assertion/blob/main/index.js#L153-L160. How can we test this?

This is an example of what the proof and proofPurpose looks like:  

```
{
  type: 'DataIntegrityProof',
  created: '2022-09-27T13:29:25Z',
  verificationMethod: 'did:key:z6MkiimK73pVRzGLpj869NybZQqPTGGogmdPMmBaAHJF4uZD#z6MkiimK73pVRzGLpj869NybZQqPTGGogmdPMmBaAHJF4uZD',
  cryptosuite: 'eddsa-2022',
  proofPurpose: 'assertionMethod',
  proofValue: 'z28qVMJ9ph8DPpbK4fGoK51DYVoU1rR41T6EqZoW9AZtpUPWjWYuq94eaacGt7eSxTxna1qy1agXzT7EXUwTEsEUx'
}

// proof.proofPurpose  => 'assertionMethod'
```

**Comment by dlongley** [comment]
The solution here should be the same / similar to the one provided in #18.

**Comment by JSAssassin** [comment]
Addressed by #22
